doc/src/build/json-rpc.md

```diff
@@ -63,7 +66,7 @@ curl --location --request POST $SUI_RPC_HOST \
 --data-raw '{ "jsonrpc":"2.0", "method":"rpc.discover","id":1}'
 ```
 
-You can see an example of the discovery service in the [OpenRPC Playground](https://playground.open-rpc.org/?schemaUrl=https://raw.githubusercontent.com/MystenLabs/sui/main/crates/sui-open-rpc/spec/openrpc.json).
+You can see an example of the discovery service in the [Sui API Reference](https://docs.sui.io/sui-jsonrpc).
```

Comment:
is it this topic?
https://docs.sui.io/devnet/build/json-rpc#rpcdiscover

Reply:
> is it this topic? https://docs.sui.io/devnet/build/json-rpc#rpcdiscover

That may well be going away. Leaving for @patrickkuo and myself to discuss.

Reply:
Removed.